src/main/java/eu/xenit/gradle/alfresco/DockerAlfrescoPlugin.java

```diff
@@ -138,7 +141,15 @@ private void updateDockerFileTask(Project project1, DockerfileWithWarsTask docke
         for(WarLabelOutputTask task: outputTasks) {
             mergeWarsTask.withLabels(task);
         }
-        mergeWarsTask.setInputWars(() -> outputTasks.stream().map(WarOutputTask::getOutputWar).collect(Collectors.toList()));
+        mergeWarsTask.setInputWars(
+                () -> Stream.concat(Stream.of(baseWar.getSingleFile()),
+                        outputTasks.stream().map(WarOutputTask::getOutputWar)).collect(Collectors.toList()));
+        
+        // Mark merged war as artifact
+        final String artifactConfiguration = "artifact" + warName + "War";
+        project.getConfigurations().create(artifactConfiguration);
+        project.getArtifacts().add(artifactConfiguration, mergeWarsTask.getOutputWar(),
```

Comment:
I think a dependency on the `mergeWarTask` is missing here, it only refers to the output file. This artifact will be invalid and will not be rebuilt when it is used.

It will not work when the output war is configured on the task, because it is read here before the task can be configured from the buildscript.